contracts/governance/gov_helpers.cairo

```diff
@@ -0,0 +1,19 @@
+%lang starknet
+
+from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.math import assert_nn_le
+
+// @notice Converts the value from Int (felt) to Uint256
+// @dev Fails if value too big, otherwise returns { low: value, high: 0 }
+// @param x: Value to be converted
+// @return Value as Uint256
+func intToUint256{range_check_ptr}(
```

Comment:
in helpers.cairo there is exactly the same function... do we keep this split? or use just one?

Reply:
helpers.cairo are AMM-specific, I don't feel like refactoring it into two helper contracts because the AMM should be frozen due to the audit. I'd like not to touch it if possible. And directly importing AMM code would just be a big mess, so this seems like the best solution